docs/guides/react-server-cli.md

```diff
@@ -114,9 +114,22 @@ export default (webpackConfig) => {
 
 In the `.reactserverrc` file add an option for `webpack-config` that points to that function file and when React Server is setting up Webpack it will call your function with the result of the built in Webpack options, allowing you to make any modifications needed.
 
+### Use Custom Express Middleware
+Currently the default Express Middlewares used are compression, body-parser, cookie-parser. If you need to setup custom express middleware you can do it with a setup function.
 
+```javascript
+export default (server, reactServerMiddleware) => {
+	server.use(compression());
+	server.use(bodyParser.urlencoded({ extended: false }));
+	server.use(bodyParser.json());
+	server.use(session(options));
+	rsMiddleware(); // Must be called once or server will not start
+}
+```
 
 
+In the `.reactserverrc` file add an option for `custom-middleware-path` that points to that function file and when React Server is setting up the server it will call your function for setup rather then the default middlewares mentioned above. On the command line the option is `--custom-middleware-path`.
+
```

Comment:
I think we've generally used the camelCase variants in the config file?  Like `customMiddlewarePath`?

Not sure if either will work, or even if I'm remembering right. ;)